types/d3-axis/d3-axis-tests.ts

```diff
@@ -58,7 +59,8 @@ const canvas: HTMLCanvasElement = select<HTMLCanvasElement, any>('canvas').node(
 
 containerElement = svg;
 containerElement = g;
-// containerElement = canvas; // fails, incompatible type
+// $ExpectError
```

Comment:
Thanks for starting to use this dts-lint feature. We should start using this across the board on all the "fail" tests in D3. We had to historically comment them out (and manually check during definition updates). So let's roll with `$ExpectError`